PyEMD/significancetest.py

```diff
@@ -0,0 +1,127 @@
+from PyEMD import EMD, EEMD, CEEMDAN
+import numpy as np
+from scipy import stats
+import math
+
+
+#helper function: normalize data
+def normalize(data):
+    mean = sum(data)/len(data)
+    dmin = min(data)
+    dmax = max(data)
+    ret = np.array([])
+    for val in data:
+        ret = np.append(ret, (val-mean)/(dmax-dmin))
+    return ret
+
+#helper function: find zero-crossings in IMF, used for finding mean period of IMF
+def sign_change(data):
+    l = len(data)
+    counter=0
+    for x in range(l-1):
+        if (data[x]>0 or data[x] == 0) and data[x+1] < 0:
+            counter+=1
+    return counter
+
+#helper function: Find mean period of an IMF
+def mean_period(data):
+    s = sign_change(data)
+    if s != 0:
+        return len(data)/s
+    else:
+        return len(data)
+
+#helper function: find energy of signal/IMF
+def energy(data):
+    return sum(pow(data, 2))
+
+#helper function: find IMF significance in 'a priori' test
+def significance_apriori(energy_density, T, N, alpha):
+    k = abs(stats.norm.ppf((1-alpha)/2))
+    if (energy_density <= (-math.log(T) + k*(math.sqrt(2/N)*math.exp(math.log(T)/2)))) and (energy_density >= (-math.log(T) - k*(math.sqrt(2/N)*math.exp(math.log(T)/2)))):
+        return False
+    else:
+        return True
```

Comment:
This statement is rather long and difficult to read. Please separate individual terms into some variables with descriptive names. If possible it should be something like
```python
if (min_energy <= energy_density <= max_energy):
  return False
else
  return True
```

but discard `return True/False` and do directly

```python
return not (min_energy <= energy_density <= max_energy)
```
**Note** that `min_energy` and `max_energy` aren't necessarily the best variable names.

Reply:
done